internal/mode/static/state/graph/httproute.go

```diff
@@ -879,6 +880,61 @@ func validateFilterHeaderModifierFields(
 	return allErrs
 }
 
+func validateFilterResponseHeaderModifier(
+	validator validation.HTTPFieldsValidator,
+	responseHeaderModifier *v1.HTTPHeaderFilter,
+	filterPath *field.Path,
+) field.ErrorList {
+	if errList := validateFilterHeaderModifier(validator, responseHeaderModifier, filterPath); errList != nil {
```

Comment:
If `responseHeaderModifier` is invalid for both `validateFilterHeaderModifier` and `disallowedResponseHeaderSet`, should we include both in `allErrs`, or do we only need to return `validateFilterHeaderModifier`?

Reply:
I think we only need to return the errors from `validateFilterHeaderModifier`.

@pleshakov, does that seem right to you?

Reply:
> I think we only need to return the errors from validateFilterHeaderModifier.

Sounds reasonable.

In the check, I would check the length of errList, not that it doesn't equal to nil - just so that we don't dependent on zero-length vs nil slice differences.